nav2_bringup/params/nav2_multirobot_params_all.yaml

```diff
@@ -0,0 +1,345 @@
+amcl:
```

Comment:
Just to make sure, we've made some changes recently. Please pull in `main` in this branch so you have the latest and greatest, then re-copy+paste the `nav2_params.yaml` file contents here with your `robot_namespace` addition to make sure its in line with all the most recent changes.

Reply:
Ok. Let me check if it is based on latest one.

Reply:
I checked the difference between `nav2_param.yaml` and `nav2_multirobot_params_all.yaml`.

And there was no diffrence except replacing part.

![Screenshot from 2023-08-16 10-22-02](https://github.com/ros-planning/navigation2/assets/21001946/8db44bc1-8b0c-41f8-98c8-8e11d7f5d678)

Reply:
OK!